lib/backend/api/api.go

```diff
@@ -106,10 +104,11 @@ type Client interface {
 	// any ready to be used.
 	EnsureInitialized() error
 
-	// Perform any "backdoor" initialization required by the components
-	// used in calico/node.  This is a temporary mechanism and will be
-	// removed.
-	EnsureCalicoNodeInitialized(node string) error
+	// Clean removes Calico data from the backend datastore.  Used for test purposes.
+	Clean() error
+
+	// Close the client.
+	//Close()
```

Comment:
why is this removed? there would be goroutine leaks if we don't close the client. I don't see `clientv3.Close()` being called manually either anywhere in the code

Reply:
Actually it was never removed, because it was never implemented in the first place for v1.  I added it here, but I haven't implemented it yet since it isn't required for CRUD, and possibly not even for MVP.  I don't think we have anything that needs to close an existing client.